tests/puppeteer/README.md

```diff
@@ -0,0 +1,56 @@
+# PrestaShop Functional Tests with puppeteer
+
+## LinkChecker
+This script will prevent the 400 and 500 http error code, by crawling your back office and front office
+
+### How to install your environment
+
+```bash
+git clone https://github.com/PrestaShop/PrestaShop/
+cd tests/puppeteer/
+npm i
+```
+
+### Available command line parameters
+
+| Parameter           | Description      |
+|---------------------|----------------- |
+| URL_BO              | URL of your PrestaShop website Back Office (default to **http://localhost:8080/admin-dev/**) |
+| URL_FO              | URL of your PrestaShop website Front Office (default to **http://localhost:8080/**) |
+| LOGIN               | LOGIN of your PrestaShop website (default to **demo@prestashop.com**) |
+| PASSWD              | PASSWD of your PrestaShop website (default to **prestashop_demo**) |
+
+### Launch script
+If you want to run the links checker test you can run the script **test/linkchecker.js**
+
+#### With default values
+
+```bash
+cd test/
+node linkchecker.js
+```
+
+#### With custom values
+
+```bash
+cd test/
+URL_BO="Your_Shop_URL_BO" URL_FO="Your_Shop_URL_FO" LOGIN="Your_Login" PASSWD="Your_Password" node linkchecker.js
+```
+
+#### Run with docker
+
+```bash
+#Build image
+docker build -t puppeteer_linkchecker -f .docker/Dockerfile .
+#Run test
+docker run -e URL_BO="Your_Shop_URL_BO" -e URL_FO="Your_Shop_URL_FO" -e LOGIN="Your_Login" -e PASSWD="Your_Password" --network="host" puppeteer_linkchecker
```

Comment:
is it possible to get these values configurable on a .env file instead? We could make one for Travis and make it part of our CI.

Reply:
Having .env file include to add dotenv dependency to the project, and we choose to not add it.
We will see for travis if we really must add it.